### PR TITLE
Example workbooks for the time series academy track

### PR DESCRIPTION
**Pull request by ckurze**
## Summary of the changes / Why this is an improvement

An initial workbook in a series that will describe how to build ML-driven time series applications with CrateDB.

## Checklist

 - [X] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: topic/timeseries/timeseries-queries-and-visualization.ipynb
Comment (amotl) [inline topic/timeseries/timeseries-queries-and-visualization.ipynb:1]: It looks like this file is already present within the tree. Can I ask you to remove it from this patch, and submit eventual updates seperately?

https://github.com/crate/cratedb-examples/blob/main/topic/timeseries/explore/timeseries-queries-and-visualization.ipynb

File: topic/timeseries/academy/batch_import_sample_weather_data_dask.ipynb
Comment (amotl) [inline topic/timeseries/academy/batch_import_sample_weather_data_dask.ipynb:1]: Despite this notebook might be targeted to the academy track, please don't introduce a separate subfolder. We can't map out the whole ontology on all axes into the filesystem hierarchy. "Flat" style is always preferred to "nested" style. Thanks!

NB: Saying that, I am also +1 on stripping the `explore` folder from the path to the other notebook. It's just nonsense.

**Review by amotl — CHANGES_REQUESTED** [review]
Thanks for submitting this. I did not do a thorough review yet, but I have two requests about layout/structure.

File: topic/timeseries/academy/batch_import_sample_weather_data_dask.ipynb
Comment (ckurze) [reply topic/timeseries/academy/batch_import_sample_weather_data_dask.ipynb:1]: Agreed! Let's wait with the removal of "explore" after our event this week, we already have some printed materials referencing it.

Comment (amotl) [reply topic/timeseries/academy/batch_import_sample_weather_data_dask.ipynb:1]: PRINTED!? ;]

Comment (amotl) [reply topic/timeseries/academy/batch_import_sample_weather_data_dask.ipynb:1]: I am fine with waiting for dissolving the `explore` folder. But can the `acadamy` folder be omitted from the very beginning, in order not to make the same mistakes, if you agree?

**Comment by amotl** [comment]
Hi. Because conflicts on this branch haven't been able to be resolved properly, the new notebook has been broken out into a different PR.

- GH-293

There is also a followup patch for that, adding a few copy-edits.

- GH-294

Thus, I am closing this PR. Feel free to re-open if you think anything has been forgotten.

**Pull request closed without merging**